### ISSUE TITLE
Skip test_xml_etree under coverage

### ISSUE THREAD

**Issue by kushaldas** [python/cpython#74627]
BPO | [30442](https://bugs.python.org/issue30442)
--- | :---
Nosy | @vstinner, @berkerpeksag, @kushaldas
PRs | <li>python/cpython#1767</li><li>python/cpython#3363</li><li>python/cpython#3549</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/kushaldas'
closed_at = <Date 2017-11-01.10:20:24.450>
created_at = <Date 2017-05-23.18:02:42.510>
labels = ['3.7', 'type-bug', 'tests']
title = 'Skip test_xml_etree under coverage'
updated_at = <Date 2017-11-01.10:20:24.447>
user = 'https://github.com/kushaldas'
```

bugs.python.org fields:
```python
activity = <Date 2017-11-01.10:20:24.447>
actor = 'berker.peksag'
assignee = 'kushal.das'
closed = True
closed_date = <Date 2017-11-01.10:20:24.450>
closer = 'berker.peksag'
components = ['Tests']
creation = <Date 2017-05-23.18:02:42.510>
creator = 'kushal.das'
dependencies = []
files = []
hgrepos = []
issue_num = 30442
keywords = ['patch']
message_count = 5.0
messages = ['294271', '294375', '301412', '302125', '305365']
nosy_count = 3.0
nosy_names = ['vstinner', 'berker.peksag', 'kushal.das']
pr_nums = ['1767', '3363', '3549']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue30442'
versions = ['Python 2.7', 'Python 3.6', 'Python 3.7']
```

</p></details>

**Comment by kushaldas** [comment]
```pytb
test test_xml_etree failed -- Traceback (most recent call last):
  File "/home/travis/build/python/cpython/Lib/test/test_xml_etree.py", line 1553, in test_bug_xmltoolkit63
    self.assertEqual(sys.getrefcount(None), count)
AssertionError: 522146 != 522143
```


Under coverage (in our travis CI) test_xml_etree fails for refcount. Brett suggests to skip this test in case running under coverage.

**Comment by kushaldas** [comment]
New changeset 1de4705d00168afa8c5b6741af02e21fc609af58 by Kushal Das in branch 'master':
bpo-30442: Skips refcount test in test_xml_etree under coverage (bpo-1767)
https://github.com/python/cpython/commit/1de4705d00168afa8c5b6741af02e21fc609af58

**Comment by vstinner** [comment]
New changeset 9a3b3852afe61f9083d336c4394984d0e92799a3 by Victor Stinner in branch '3.6':
bpo-30442: Skips refcount test in test_xml_etree under coverage (bpo-1767) (bpo-3363)
https://github.com/python/cpython/commit/9a3b3852afe61f9083d336c4394984d0e92799a3

**Comment by vstinner** [comment]
New changeset 860839cc8e0a4a0890418f77c984955697f96828 by Victor Stinner (Miss Islington (bot)) in branch '2.7':
[2.7] bpo-30442: Skips refcount test in test_xml_etree under coverage (GH-1767) (bpo-3549)
https://github.com/python/cpython/commit/860839cc8e0a4a0890418f77c984955697f96828

**Comment by berkerpeksag** [comment]
All PRs have been merged and 3.5 is now in security-fix-only mode. Closing this as 'fixed'.
